app/src/main/java/com/veriff/demo/screens/login/LoginMVP.kt

```diff
@@ -0,0 +1,17 @@
+package com.veriff.demo.screens.login
+
+import com.veriff.demo.base.VeriffFlowMVP
+import com.veriff.demo.base.VeriffFlowModel
+
+interface LoginMVP : VeriffFlowMVP {
+    interface View : VeriffFlowMVP.View {
+        fun showEmailError(msg: String)
+        fun showPasswordError(msg: String)
+        fun showLoginError(msg: String)
+    }
+
+    abstract class Presenter(view: View, model: LoginModel)
```

Comment:
I'm still arguing to keep the number of abstract classes/interfaces as low as possible and have this abstract class in actual implementation already. Then remove the LoginMVP and only LoginView interface will remain as this is needed for presenter - abstract Presenter is not needed for anything